scripts/run_bp.py

```diff
@@ -155,20 +165,79 @@ def check_environment():
                                'system time')
 
 
+def github_get_commits(path):
+    return '%s/repos/%s/%s/commits?path=%s&sha=%s' % (_GITHUB_API, _SCRIPT_ORG, _SCRIPT_REPO, path, _SCRIPT_BRANCH)
```

Comment:
please check 80 line alignment for all changes

Reply:
also the `sha=%s` 's argument is `_SCRIPT_BRANCH`?

Reply:
`sha` parameter takes branch as well.